### PR TITLE
Replace Sublime Text with "Text Editor"

### PR DESCRIPTION
**Pull request by db0sch**
Changes a bit the wording to make it a bit more generic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Memo for my beloved students
 - [Keyboard's shortcuts (Mac)](#keyboards-shortcuts-mac)
   - [Special characters](#special-characters)
   - [Navigate](#navigate)
-  - [Sublime Text](#sublime-text)
+  - [Text Editor](#sublime-text)
```
Comment (Eschults) [inline README.md]: Let's update the anchors too 👌

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ from one program to another                # cmd + tab
 from one window to another (same program)  # cmd + `
 ```
 
-### Sublime Text
+### Text Editor (Sublime Text or VS Code with Sublime Text Keymap)
```
Comment (Eschults) [inline README.md]: Should we leave only "Text Editor" as the title and add a `<p>` to specify both environments ❓

**Review by Eschults — COMMENTED** [review]
Good catch @db0sch 🙌

File: README.md (hunk 1 above)
Comment (db0sch) [reply README.md]: ✅

**Comment by Eschults** [comment]
👍

**Pull request merged**